SKILL.tsv

```diff
@@ -3785,10 +3785,10 @@ SKILL_20150414_003784	Cures a poisoned ally. A poison status with a higher level
 SKILL_20150414_003785	{#DD5500}{ol}[Physical] - [Poison] - [Pierce]{/}{/}{nl}Blow toxic arrow and inflict continuous damage.
 SKILL_20150414_003786	{#DD5500}{ol}[Physical] - [Poison] - [Pierce]{/}{/}{nl}Throw jar of Poison to create poisonous puddle. Enemies become poisoned when stepping on the puddle.
 SKILL_20150414_003787	{#DD5500}{ol}[Physical] - [Poison] - [Pierce]{/}{/}{nl}Throw poison to inject a parasite into the enemy. Poisonous insects are born when the host is killed.
-SKILL_20150414_003788	Hide yourself in an empty barrel to prevent attention from enemies. In this state, you are unaffected to a few physical attacks.
+SKILL_20150414_003788	Hide yourself in an empty barrel to prevent attention from enemies. In this state, you are unaffected to a certain amount of physical attacks.
```

Comment:
"In this state, you are unaffected to a certain amount of physical attacks."
i think it should be "unaffected by a certain amount" rather than "to," since you're generally affected by something rather than affected to something